src/components/Person/Person.jsx

```diff
@@ -1 +1,38 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  let familyStatus = 'wife';
+
+  if (sex === 'f') {
+    familyStatus = 'husband';
+  }
```

Comment:
I think you can combine these lines using the ternary operator and get rid of 'if'

Reply:
agree, it would be better